### PR TITLE
AEA-1571 Add sandbox EPSAT environment with singular dose to text placeholder page

### PR DESCRIPTION
**Pull request by booshibooshi**

### CODE REVIEW

File: tool/site/server/src/services/communication/signing-client.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export function getSigningClient(request: Hapi.Request, accessToken: string): Si
   return (isDev(CONFIG.environment) && getSessionValue("use_signing_mock", request))
     || (isQa(CONFIG.environment) && getSessionValue("use_signing_mock", request))
     || isLocal(CONFIG.environment)
+    || isSandbox(CONFIG.environment)
```
Comment (booshibooshi) [inline tool/site/server/src/services/communication/signing-client.ts:21]: defunct change, to remove

File: tool/site/server/src/app.ts [hunk 2]
```diff
@@ -45,25 +47,27 @@ function createServer() {
...
-      isSecure: true
-    },
-    redirectTo: (request: Hapi.Request) => {
-      if (isDev(CONFIG.environment)) {
-        setSessionValue(
-          "use_signing_mock",
-          request.query["use_signing_mock"],
-          request
-        )
+  if (!isSandbox(CONFIG.environment)) {
```
Comment (JackRobinsonIW) [inline tool/site/server/src/app.ts:50]: with the above change (lines 18-20) this function is only called when not in sandbox mode, so not sure we need this check here?

File: tool/site/server/src/app.ts [hunk 3]
```diff
@@ -132,23 +136,30 @@ function addApiRoutes(server: Hapi.Server) {
...
+    server.route(addView("prescribe/cancel"))
+    server.route(addView("dispense/release"))
+    server.route(addView("dispense/verify"))
+    server.route(addView("dispense/return"))
+    server.route(addView("dispense/dispense"))
+    server.route(addView("dispense/withdraw"))
+    server.route(addView("dispense/claim"))
+  }
+
+  if (isDev(CONFIG.environment) || isLocal(CONFIG.environment) || isQa(CONFIG.environment)) {
```
Comment (JackRobinsonIW) [inline tool/site/server/src/app.ts:160]: I like this change

**Review by JackRobinsonIW — DISMISSED** [review]

File: tool/site/server/src/app.ts (hunk 2 above)
Comment (booshibooshi) [reply tool/site/server/src/app.ts:50]: Good spot, forgot I'd already made that change some time ago. Will remove duplication

**Pull request merged**